packages/oid4vci-demo-frontend/public/locales/development/nl/translation.json

```diff
@@ -66,5 +66,7 @@
   "title_secure_your_credential": "Bewaar uw {{credentialName}} credential",
   "title_secure_your_personal_credential": "Bewaar uw persoonlijke credential",
   "paragraph_add_credential": "Laten we de credential aan uw wallet toevoegen.",
-  "paragraph_scan_this_qr": "Scan deze QR-code met de QR-scanner van uw wallet. Als u al op uw telefoon bent, open dan uw wallet door op de knop 'Openen in wallet' te klikken."
+  "paragraph_scan_this_qr": "Scan deze QR-code met de QR-scanner van uw wallet. Als u al op uw telefoon bent, open dan uw wallet door op de knop 'Openen in wallet' te klikken.",
+  "ssi_information_request_page_form_terms_and_conditions_label": "I agree to the",
```

Comment:
a bit trivial, maybe nicer to add this one after line 57

Reply:
i personally do not mind, but ii can get the comment. moved it up to after 57